jakartams/src/main/resources/reference.conf

```diff
@@ -0,0 +1,134 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# Settings for the Apache Pekko Connectors JMS connector
+#
+pekko.connectors.jakartams {
+  #connection-retry
+  # Connection Retry Settings
+  # these set the defaults for Consumer, Producer, and Browse settings
+  connection-retry {
+    # Time allowed to establish and start a connection.
+    connect-timeout = 10 seconds
+    # Wait time before retrying the connection the first time.
+    initial-retry = 100 millis
+    # Back-off factor for subsequent retries.
+    backoff-factor = 2
+    # Back-off factor for subsequent retries.
+    max-backoff = 1 minute
```

Comment:
the comment of max-backoff is same to backoff-factor

Reply:
Updated the comment in both jms and jakartams